sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresStringTests.java

```diff
@@ -823,7 +823,60 @@ public void testLike3Pattern() {
                  t""");
     }
 
-    // TODO ILIKE
+    @Test
+    public void testILike2() {
+        this.q("""
```

Comment:
you can combine multiple such tests using the qs function. but let's leave them right now as they are.

Reply:
Good point, we can refactor all those to qs.